Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -75,6 +77,25 @@ async function run() {
         endPoint["subscriptionId"] = tl.getEndpointDataParameter(connectedServiceName, 'subscriptionid', true);
         endPoint["envAuthUrl"] = tl.getEndpointDataParameter(connectedServiceName, 'environmentAuthorityUrl', true);
         endPoint["url"] = tl.getEndpointUrl(connectedServiceName, true);
+        endPoint["environment"] = tl.getEndpointDataParameter(connectedServiceName, 'environment', true);
```

Comment:
As discussed in person, can we push this logic within the azure stack utility you already have.